client/src/components/workspace-nav/OrganizationMenu.jsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import * as auth from './../../utilities/auth';
 
 function OrganizationMenu({ profile, authURL }) {
+  const url = authURL.includes('auth0') ? 'https://manage.auth0.com' : `${authURL}/realms/akvo/account`;
```

Comment:
`authURL.includes('auth0')` seems too brittle. Why not move this logic to the `auth` object?